aioredis/cluster/cluster.py

```diff
@@ -203,6 +169,16 @@ def get_node_by_slot(self, slot):
         else:
             return None
 
+    def get_node_by_id(self, node_id):
+        for node in self.nodes:
+            if node_id == node.id:
+                return node
```

Comment:
In my opinion, if a function returns a value in some path, it should use return statements in all paths to make it obvious that the function has a meaningful return value.